src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -209,7 +209,7 @@ namespace Microsoft::Console::VirtualTerminal
     class FlaggedEnumValue
     {
     public:
-        static constexpr VTInt mask{ WI_StaticAssertSingleBitSet(Flag) };

```

Comment:
Yeah, that was a bit confusing. I couldn't figure out why it would suddenly have a problem with that, but when `Flag` is 0 that doesn't count as a single bit. I'm assuming it didn't complain before because we hadn't actually used the ANSI modes yet.